torch/_dynamo/output_graph.py

```diff
@@ -1397,6 +1397,7 @@ def compile_subgraph(
                 overridden_sources=overridden_sources,
             )
             self.codegen_suffix(tx, stack_values_flat, pass1)
+            self.side_effects.close_local_generators()
```

Comment:
add a comment about why this is here.
also TODO(rzou): is this actually the right place?